plugins/apitracing/src/lib/ApiTracing.cpp

```diff
@@ -0,0 +1,121 @@
+#include "Config.h"
+#include "Filenames.h"
+#include "Tracer.h"
+#include "config/TracingTargetsParser.h"
+#include "os/windows/Library.h"
+#include <fmt/core.h>
+#include <tclap/CmdLine.h>
+#include <vmicore/plugins/PluginInit.h>
+#include <vmicore/plugins/PluginInterface.h>
+
+VMI_PLUGIN_API_VERSION_INFO
+
+namespace ApiTracing
+{
+    namespace
+    {
+        VmiCore::Plugin::PluginInterface* pluginInterface;
```

Comment:
I recommend using declaratives instead of fully qualified names. They are fine to use in compilation units (as opposed to headers).